skywalking-sniffer/skywalking-sdk-plugin/mongodb-plugin/src/main/java/com/a/eye/skywalking/plugin/mongodb/MongoDBMethodInterceptor.java

```diff
@@ -0,0 +1,139 @@
+        } else if (obj instanceof DistinctOperation) {
+            return new OperationInfo(ReadMethod.DISTINCT.getName(), ((DistinctOperation) obj).getFilter().toString());
+        } else if (obj instanceof FindOperation) {
+            return new OperationInfo(ReadMethod.FIND.getName(), ((FindOperation) obj).getFilter().toString());
+        } else if (obj instanceof GroupOperation) {
+            return new OperationInfo(ReadMethod.GROUP.getName(), ((GroupOperation) obj).getFilter().toString());
+        } else if (obj instanceof ListCollectionsOperation) {
+            return new OperationInfo(ReadMethod.LIST_COLLECTIONS.getName(), ((ListCollectionsOperation) obj).getFilter().toString());
+        } else if (obj instanceof MapReduceWithInlineResultsOperation) {
+            return new OperationInfo(ReadMethod.MAPREDUCE_WITHINLINE_RESULTS.getName(), ((ListCollectionsOperation) obj).getFilter().toString());
+        } else  if (obj instanceof DeleteOperation) {
+            return new OperationInfo(WriteMethod.DELETE.getName(), ((DeleteOperation) obj).getDeleteRequests().toString());
+        } else if (obj instanceof InsertOperation) {
+            return new OperationInfo(WriteMethod.INSERT.getName(), ((InsertOperation) obj).getInsertRequests().toString());
+        } else if (obj instanceof UpdateOperation) {
+            return new OperationInfo(WriteMethod.UPDATE.getName(), ((UpdateOperation) obj).getUpdateRequests().toString());
+        } else if (obj instanceof CreateCollectionOperation) {
+            return new OperationInfo(WriteMethod.CREATECOLLECTION.getName(), ((CreateCollectionOperation) obj).getCollectionName());
+        } else if (obj instanceof CreateIndexesOperation) {
+            return new OperationInfo(WriteMethod.CREATEINDEXES.getName(), ((CreateIndexesOperation) obj).getIndexNames().toString());
+        } else if (obj instanceof CreateViewOperation) {
+            return new OperationInfo(WriteMethod.CREATEVIEW.getName(), ((CreateViewOperation) obj).getViewName());
+        } else if (obj instanceof FindAndDeleteOperation) {
+            return new OperationInfo(WriteMethod.FINDANDDELETE.getName(), ((FindAndDeleteOperation) obj).getFilter().toString());
+        } else if (obj instanceof FindAndReplaceOperation) {
+            return new OperationInfo(WriteMethod.FINDANDREPLACE.getName(), ((FindAndReplaceOperation) obj).getFilter().toString());
+        } else if (obj instanceof FindAndUpdateOperation) {
+            return new OperationInfo(WriteMethod.FINDANDUPDATE.getName(), ((FindAndUpdateOperation) obj).getFilter().toString());
+        } else if (obj instanceof MapReduceToCollectionOperation) {
+            return new OperationInfo(WriteMethod.MAPREDUCETOCOLLECTION.getName(), ((MapReduceToCollectionOperation) obj).getFilter().toString());
+        } else if (obj instanceof MixedBulkWriteOperation) {
```

Comment:
The bulk api will be transfer large data, make sure about you need every document in this bulk operate.